### ISSUE TITLE
Apply Rich Text Standards for Content Types

### ISSUE THREAD

**Issue by budparr** [theNewDynamic/hugo-module-tnd-seo#4]
This notice came for thenewdynamic.org but it's a signal that Google expects fields in JSON+LD to conform to rich text standards, and I think, of the various standards, events is one we should account for and we should review all content types for logical inclusion

https://developers.google.com/search/docs/data-types/event

<details><summary>Details</summary>

> Search Console has identified that your site is affected by 6 Events issues:

> Top Errors
> Errors can prevent your page or feature from appearing in Search results. The following errors were found on your site:
> Missing field "startDate"
> Missing field "name"
> Missing field "location"

> Top Warnings
> Warnings are suggestions for improvement. Some warnings can affect your appearance on Search; 
 some might be reclassified as errors in the future. The following warnings were found on your site:
> Missing field "endDate"
> Missing field "performer"
> Missing field "offers"

</details>

**Comment by regisphilibert** [comment]
Yes. It's a bit tricky though. Directories (and therefor content types) are rarely named the same across multiple users. `events`, `event` etc... Plus the Front Matter keys needed to build say the "location" map of an event will also never be consistent across multiple users.

I suppose the idea, is the add another form of extendability limited to jsonld data per types so users can build whatever they need for it.

**Comment by budparr** [comment]
Well we can standardize for users, just in the way Google is doing. If you have events, you need X field and Y field, etc. And if you do that we'll give you the output to make Google happy. Am I wrong in thinking that's appropriate? 

Ultimately, this is an important feature because Google will send end-users Error messages. I'd rather it be a bit rigid than not have it or try to make something that can handle every case out there
